### ISSUE TITLE
Add feature to update command to specify artist

### ISSUE THREAD

**Issue by jessicando** [fmbot-discord/fmbot#218]
**What command would you like to see improved or added?**
Add feature to the update command to specify an artist.

**What exactly should it do?**
Allow users to specify an artist for the update command. When this artist is specified the update command should look at only those scrobbles already tagged under the artist for full cache updates updates

**Additional context**
As a long-time last.fm user with >400k scrobbles, I have a fair number of artists with messy tags.  I often update one specific artist's tags at a time when I listen to them again. Presently a full update can feel wasteful and have long run times - especially as an .fmbot supporter with full storage. I don't imagine I'm the only one with this use case.

**Comment by th0mk** [comment]
Good idea. You can also already run the `.topartists alltime` command, this will also update the playcounts for all artists in there.
